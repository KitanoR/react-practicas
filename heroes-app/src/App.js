import logo from './logo.svg';
import { AppRouter } from './routers/AppRouter';

function App() {
  return (
    <AppRouter />
  );
}

export default App;
