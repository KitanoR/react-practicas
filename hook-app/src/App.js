import React from 'react'

export default function HooksApp() {
  return (
    <div>
      Hola mundo
    </div>
  )
}
