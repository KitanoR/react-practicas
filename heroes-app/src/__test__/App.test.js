import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Test para el inicial', () => {
    test('Validando montar el test ', () => {
        const wrapper = shallow(<App />)
        expect(wrapper).toMatchSnapshot();
    });
});