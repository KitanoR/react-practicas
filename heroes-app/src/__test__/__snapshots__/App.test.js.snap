// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test para el inicial Validando montar el test  1`] = `<AppRouter />`;
